engine/script/src/script_hash.cpp

```diff
@@ -214,15 +249,14 @@ namespace dmScript
         }
 
         int ref = *refp;
-
         // Retrieve the value from the strong table
         lua_rawgeti(L, LUA_REGISTRYINDEX, context->m_ContextTableRef);
         // [-1] context_table
         lua_rawgeti(L, -1, ref);
         // [-2] context_table
         // [-1] value
 
-        if (lua_isnil(L, -1))
+        if (!lua_isuserdata(L, -1))
```

Comment:
it may be nil or a number, because that's how registry work (something like a linked list of empty slots)